src/tests/topp/TextExporter_5_output_peptides.txt

```diff
@@ -1,6 +1,6 @@
-#rt	mz	score	rank	sequence	charge	aa_before	aa_after	score_type	search_identifier	accessions	predicted_rt	rt_first_dim	predicted_rt_first_dim	predicted_pt
-1234.5	675.9	0	0	TC(Carboxymethyl)VADESHAGC(Carboxymethyl)EK	2	K	S	MOWSE	Mascot_2006-01-12T12:13:14_5233264595117471314	PROT1;PROT2	359.970709999999997	4.0	0.51502823	-1
-1234.5	675.9	0	0	KC(Carboxymethyl)PSPC(Carboxymethyl)QLQAER	2	K	A	MOWSE	Mascot_2006-01-12T12:13:14_5233264595117471314	PROT1	892.753230000000031	4.0	4.7040273	-1
--1	-1	0	0	KC(Carboxymethyl)PSPC(Carboxymethyl)QLQAER	2	K	A	MOWSE	Mascot_2006-01-12T12:13:14_5233264595117471314	PROT2	892.753230000000031	6.0	4.7040273	-1
--1	-1	0.001507159	0	KC(Carboxymethyl)PSPC(Carboxymethyl)QLQAER	2	K	A	MOWSE	Mascot_2006-01-12T12:13:14_5233264595117471314	PROT2	892.753230000000031	6.0	4.7040273	-1
--1	-1	0	0	EYEATLEEC(Carboxymethyl)C(Carboxymethyl)AK	2	K	D	MOWSE	Mascot_2007-01-12T12:13:14_4835329514588776807	PROT1	900.572959999999966	6.0	3.9354399	-1
+#rt	mz	score	rank	sequence	charge	aa_before	aa_after	score_type	search_identifier	accessions	start	end	predicted_rt	rt_first_dim	predicted_rt_first_dim	predicted_pt
+1234.5	675.9	0	0	TC(Carboxymethyl)VADESHAGC(Carboxymethyl)EK	2	K	S	MOWSE	Mascot_2006-01-12T12:13:14_5233264595117471314	PROT1;PROT2	;	;	359.970709999999997	4.0	0.51502823	-1
```

Comment:
There seem to be an issue that ; is printed even if the information is not present

Reply:
I intended this, in case there exists start/end information for only some of the accessions. You would need to have some kind of indication which accession these belong to.

Reply:
Sounds good! Did not think about that

Reply:
Now it only prints redundant semincolons if there is information at all

Reply:
@cbielow  looks good from my side. Should we do something about the number of digits for the score?

Reply:
we should solve this at a higher level. I remember @jpfeuffer fixing some precision issues, but from what I remember boost does not allow fine control over the precision (only the number of digits, which is not the same thing)